File: python/random_3/random.py
import random


def generate_number():
    return random.randint(10, 99)


number = generate_number()
reversed_number = str(number % 10) + str(int(number / 10))

print(f'Wylosowano liczbę: {number}')
print(f'Odwórcona wylosowana liczba ma postać: {reversed_number}')

File: javascript/random_3/random.js
const generateRandomNumber = (from, to) => {
    return Math.floor(Math.random() * (to - from) + from)
}

const changeDigitsInNumber = (number) => {
    console.log(`Wylosowano liczbę: ${number}`)
    const numberArray = number.toString().split('')
    return numberArray[1] + numberArray[0]
}

console.log(`Odwrócona liczba ma postać: ${changeDigitsInNumber(generateRandomNumber(10, 99))}`)
